book/src/database-migrations.md

```diff
@@ -16,6 +16,7 @@ validator client or the slasher**.
 
 | Lighthouse version | Release date | Schema version | Downgrade available? |
 |--------------------|--------------|----------------|----------------------|
+| v5.2.0             | Jun 2024     | v19            | yes                  |
```

Comment:
```suggestion
| v5.2.0             | Jun 2024     | v19            | yes before Deneb     |
```

Reply:
the schema actually can't be downgraded any more, I'm planning to open a PR to delete the old schema migrations once the v20 PR is merged